packages/extension-api/src/extension-api.d.ts

```diff
@@ -1834,6 +1858,70 @@ declare module '@podman-desktop/api' {
     export const clipboard: Clipboard;
   }
 
+  /**
+   * Represents a dictionary object with string keys and values of type T.
+   */
+  export interface Dict<T> {
+    /**
+     * Index signature for accessing values by keys.
+     */
+    [key: string]: T | undefined;
+  }
+
```

Comment:
I would avoid to introduce a dictionary

custom env can just be `{ [key: string]: string };`

Reply:
removed the interface and updated `RunOptions`